feat(android): Add docs for screenshots/vh BeforeCaptureCallback (#7239)
getsentry/sentry-docs: src/platform-includes/enriching-events/attach-viewhierarchy/android.mdx

</application>
```

### Customize View Hierarchy Capturing

<Note>

Requires SDK version `6.24.0` or higher.

</Note>

> On Android capturing view hierarchy can be an resource-intense operation, thus capturing is limited to one view hierarchy every 2 seconds using a debouncing mechanism. This behavior can be overruled by supplying a `BeforeCaptureCallback` for view hierarchies in the `SentryAndroidOptions`.
> 

Reviewer: ```suggestion
Because capturing view hierarchy can be a resource-intensive operation on Android, it's limited to one view hierarchy every 2 seconds using a debouncing mechanism. This behavior can be overruled if you supply a `BeforeCaptureCallback` for view hierarchies in the `SentryAndroidOptions`.

```

> The `BeforeCaptureCallback` also allows you to customize the behavior based on event data, giving you full control over when a view hierarchy should be captured and when not.
> 
> For example, view hierarchy capturing can be limited to only crashed events:

Reviewer: ```suggestion
The `BeforeCaptureCallback` also allows you to customize behavior based on event data so you can decide when to capture view hierarchy and when not to. For example, you can decide to only capture view hierarchy for crashed events:
```
